docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -1,67 +1,172 @@
-== Getting started
+== Getting Started
 
-There are many available resources to get you up to speed with our libraries as quickly as possible.
+This section describes how to get up to speed with Spring Cloud GCP libraries.
 
-=== Spring Initializr
+=== Setting up Dependencies
 
-There are three entries in http://start.spring.io/[Spring Initializr] for Spring Cloud GCP.
+All Spring Cloud GCP artifacts are made available through Maven Central.
+The following resources are provided to help you setup the libraries for your project:
 
-==== GCP Support
+- Maven Bill of Materials for dependency management
+- Starter Dependencies for depending on Spring Cloud GCP modules
 
-The GCP Support entry contains auto-configuration support for every Spring Cloud GCP integration.
-Most of the autoconfiguration code is only enabled if other dependencies are added to the classpath.
+You may also consult our https://github.com/spring-cloud/spring-cloud-gcp[Github project] to examine the code or build directly from source.
+
+==== Bill of Materials
+
+The Spring Cloud GCP Bill of Materials (BOM) contains the versions of all the dependencies it uses.
+
+If you’re a Maven user, adding the following to your pom.xml file will allow you omit any Spring Cloud GCP dependency version numbers from your configuration.
+Instead, the version of the BOM you’re using determines the versions of the used dependencies.
+
+[source,xml,subs="normal"]
+----
+<dependencyManagement>
+   <dependencies>
+       <dependency>
+           <groupId>org.springframework.cloud</groupId>
+           <artifactId>spring-cloud-gcp-dependencies</artifactId>
+           <version>{project-version}</version>
+           <type>pom</type>
+           <scope>import</scope>
+       </dependency>
+   </dependencies>
+</dependencyManagement>
+----
+
+In the following sections, it will be assumed you are using the Spring Cloud GCP BOM and the dependency snippets will not contain versions.
+
+Gradle users can achieve the same kind of BOM experience using Spring's https://github.com/spring-gradle-plugins/dependency-management-plugin[dependency-management-plugin] Gradle plugin.
+For simplicity, the Gradle dependency snippets in the remainder of this document will also omit their versions.
+
+==== Starter Dependencies
+
+Spring Cloud GCP offers https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-starters[starter dependencies] through Maven to easily depend on different modules of the library.
+Each starter contains all the dependencies and transitive dependencies needed to begin using their corresponding Spring Cloud GCP module.
+For example, if you wish to write a Spring application with Cloud Pub/Sub, you would include the `spring-cloud-gcp-starter-pubsub` dependency in your project.
```

Comment:
```suggestion
For example, if you wish to write a Spring application with Cloud Pub/Sub, you would include the `spring-cloud-gcp-starter-pubsub` dependency in your project. You do *not* need to include the underlying  `spring-cloud-gcp-pubsub` dependency, because the `starter` dependency includes it.
```

Reply:
Done.